recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch

```diff
@@ -0,0 +1,25 @@
+From 1ba241438dcc14852017757451521f5e27690290 Mon Sep 17 00:00:00 2001
+From: Cristinel Panfir <cristinel.panfir@nxp.com>
+Date: Mon, 2 May 2022 15:18:46 +0000
+Subject: [PATCH] Findtim-vx.cmake: Fix LIBDIR for multilib environment
+
```

Comment:
Upstream-Status field, please.

Reply:
What do you mean by this? I pulled patch directly from NXP BSP:

https://source.codeaurora.org/external/imx/meta-imx/tree/meta-ml/recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch?h=kirkstone-5.15.32-2.0.0

Reply:
great. But we need to have the header. See:

https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status